ToolBox/AlgoSeekOptionsConverter/Utilities.cs

```diff
@@ -0,0 +1,62 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Diagnostics;
+using Ionic.BZip2;
+using System.IO;
+using System.Threading.Tasks;
+using QuantConnect.Logging;
+using QuantConnect.Data;
+using QuantConnect.Util;
+
+
+namespace QuantConnect.ToolBox.AlgoSeekOptionsConverter
+{
+	public static class Extensions
+	{
+		public static AlgoSeekOptionsReader EarliestTick(this IEnumerable<AlgoSeekOptionsReader> readers)
```

Comment:
This whole Utilties.cs can be removed really. 
+ The EarliestTick is too specific // only used in 1 place.
+ When you remove the benchmark logging the Info is only used in 1 place as well,
+ Resolutions is used a few places but really should be solved in a loop: so points to a design flaw
```
     foreach(var resolution in X) process.AddConsolidators(new Consolidator(resolution));
```